src/api/app/models/package.rb

```diff
@@ -224,12 +219,8 @@ def self.exists_by_project_and_name(project, package, opts = {})
   end
 
   def self.exists_on_backend?(package, project)
-    begin
-      answer = Backend::Connection.get(Package.source_path(project, package))
-      return true if answer
-    rescue Backend::Error
-      # ignored
-    end
+    !Backend::Connection.get(Package.source_path(project, package)).nil?
```

Comment:
I was normally used to use the `!!` technique to make every value boolean but Rubocop doesn't like. Instead it recommends https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/DoubleNegation instead